app/assets/javascripts/discourse/app/components/bread-crumbs.hbs

```diff
@@ -1,3 +1,18 @@
+<PluginOutlet
+  @name="bread-crumbs-left"
```

Comment:
Maybe…

```suggestion
  @name="bread-crumbs-before"
```

…would be better? I'm thinking of RTL langs, weird vertical breadcrumb in themes, etc.

Reply:
Ah yes, I prefer `before` as well, but wasn't sure if I should say before or left to be consistent with the other plugin outlet that appears after:
```
<PluginOutlet
  @name="bread-crumbs-right"
```

What do you think? I'm fine with either..

Reply:
Ah, well, that settles it then 😅 left & right it is 😛